pkg/kmsplugin/kms.go

```diff
@@ -62,7 +62,7 @@ func ParseError(err error) (errorType KMSErrorType) {
 	zap.L().Debug("parsed error", zap.String("code", ae.ErrorCode()), zap.String("message", ae.ErrorMessage()))
 	var defaultCodes retry.IsErrorThrottles = retry.DefaultThrottles
 	if defaultCodes.IsErrorThrottle(uerr) == aws.TrueTernary {
-		return KMSErrorTypeThrottled
```

Comment:
not sure what else the throttled error type is for, but we could also change livez to not fail if it's user-induced or throttled, instead of just user-induced.